bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -107,6 +107,7 @@ spec:
         # TODO: remove
         - mountPath: /var/run
           name: host-var-run
+          mountPropegation: Bidirectional
```

Comment:
Shouldn't it be mountPropagation?

https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation

Reply:
...yes, thank you

Reply:
I knew I should copy and paste it, spelling is hard :upside_down_face: fixed!